snmp/README.md

```diff
@@ -33,7 +33,7 @@ To use Autodiscovery with the SNMP check:
 
 | Parameter                    | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
 |------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `profiles`                   | A list of profiles to use. A profile is a collection of OIDs the Datadog Agent collects metrics and associated tags from. A complete list of Datadog supported profiles can be found in [Github][5]. Profiles can be referenced by file, under `definition_file`, or written inline under `definition`. Any of the OOTB Datadog profiles can be listed by their name. Additional custom profiles can be referenced by the file path. **Note**: The generic profile is `generic_router.yaml`, which should work for routers, switches, etc. |
+| `profiles`                   | A list of profiles to use. A profile is a collection of OIDs the Datadog Agent collects metrics and associated tags from. A complete list of Datadog supported profiles can be found in [Github][5]. Profiles can be referenced by file, under `definition_file`, or written inline under `definition`. Any of the OOTB Datadog profiles can be listed by their name. Additional custom profiles can be referenced by the file path, or simply dropped in the configuration directory. By default, all profiles shipped by the agent and in the configuration directory are loaded. **Note**: The generic profile is `generic_router.yaml`, which should work for routers, switches, etc. |
```

Comment:
Since this is now an optional part of the config, it should more clearly indicate what adding values here means and that custom profiles don't need to be listed unless the goal is to specifically limit the active profiles.  Something like this:

```suggestion
| `profiles`                   | A list of profiles to use. A profile is a collection of OIDs the Datadog Agent collects metrics and associated tags from and a complete list of Datadog supported profiles can be found in [Github][5]. By default, all profiles shipped by the agent and in the configuration directory are loaded.  To customize the specific profiles for collection, they can be explicitly referenced by filename under `definition_file`, or written inline under `definition`. Any of the OOTB Datadog profiles can be listed by their name. Additional custom profiles can be referenced by the file path in the config, or simply dropped in the configuration directory. **Note**: The generic profile is `generic_router.yaml`, which should work for routers, switches, etc. |
```